jerry-core/ecma/builtin-objects/ecma-builtin-global.c

```diff
@@ -100,31 +100,32 @@ ecma_builtin_global_object_print (ecma_value_t this_arg __attr_unused___, /**< t
 
       if (code_point == LIT_CHAR_NULL)
       {
-        printf ("\\u0000");
+        jerry_port_putchar ('\0');
```

Comment:
As discussed in #945, this is incorrect. The original code prints 6 characters (the backslash, the small u, and four zero digits), while the suggested change would print the NUL character (i.e., a byte of 0).